imap-proto/src/parser/rfc3501.rs

```diff
@@ -47,7 +48,7 @@ named!(status<Status>, alt!(
     status_bye
 ));
 
-named!(mailbox<&str>, map!(
+named!(pub(crate) mailbox<&str>, map!(
```

Comment:
Why make this `pub(crate)`? That seems unrelated to the main goal of this PR.

Reply:
There was some reason for making it pub(crate), but it looks like the reason has gone during development.